config/registry.yaml

```diff
@@ -121,8 +121,12 @@ modules:
       linkerd_high_availability: optional
       linkerd_enabled: optional
       delegated: optional
-      hosted_zone_name: optional
-    outputs: {}
+      private_key: optional
+      certificate_body: optional
+      certificate_chain: optional
```

Comment:
Why do we need a cert chain? Isn't that a rare case?

Reply:
I don't think I've ever seen a certificate trusted by a root CA of the internet and unless you're some organization in charge of one of these CAs in which case wow.

Reply:
Ah I see. It's fine then.